### DISCUSSION TITLE
Gamepads for android is compatible With BlueRetro GC? ?

### DISCUSSION THREAD

**Discussion by JackLine087** [darthcloud/BlueRetro#232] [Q&A] [answered]
Is it compatible with android bluetooth controls like this t3 and other gamepads bluetooth for android in BlueRetro?

![H42c5dc0a4ad549fa8e76bbc31804c815Y](https://user-images.githubusercontent.com/101960991/159148554-7b91c6fb-3c55-4f87-bafb-51fa02c33939.jpg)

**Comment by darthcloud** [comment, marked as answer]
List of well supported controller here:
https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide

Random controller might work or might not. If it work it will likely have weird buttons mapping.
